widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -161,6 +162,9 @@ export function Quote(props: QuoteProps) {
               USD_VALUE_MIN_DECIMALS,
               USD_VALUE_MAX_DECIMALS
             ),
+            realValue: index === 0 ? input.value : swap.fromAmount,
+            realUsdValue:
+              (swap.from.usdPrice ?? 0) * parseFloat(swap.fromAmount),
```

Comment:
Using BigNumber.js is recommended for precision and consistency.

Reply:
fixed.